lib/ansible/plugins/filter/combine.yml

```diff
@@ -20,25 +21,36 @@ DOCUMENTATION:
       type: bool
       default: false
     list_merge:
-      describe: Behaviour when encountering list elements.
+      description: |
+        Behaviour when encountering list elements.
+
+        + `replace`: overwrite older entries with newer ones
+        + `keep`: discard newer entries
+        + `append`: append newer entries to the older ones
+        + `prepend`: insert newer entries in front of the older ones
+        + `append_rp`: append newer entries to the older ones, overwrite duplicates
+        + `prepend_rp`: insert newer entries in front of the older ones, discard duplicates
```

Comment:
A general remark: do not try to use MarkDown or RST formatting directly in docs. The result will not look good. See https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#linking-and-other-format-macros-within-module-documentation for the formatting directives allowed.

Reply:
Thanks Felix, good to know!

Reply:
describe/description aside, this is not the desired change, the config system has supported description in choices, fixes will be happening on the html docs building side to generate this page.